api/client/client.go

```diff
@@ -828,23 +828,21 @@ func (c *Client) GetKubeServices(ctx context.Context) ([]types.Server, error) {
 
 // GetApplicationServers returns all registered application servers.
 func (c *Client) GetApplicationServers(ctx context.Context, namespace string) ([]types.AppServer, error) {
-	resp, err := c.grpc.GetApplicationServers(ctx, &proto.GetApplicationServersRequest{
-		Namespace: namespace,
-	}, c.callOpts...)
+	resources, err := c.ListAllResources(ctx, namespace, proto.ResourceType_RESOURCE_TYPE_APPLICATION_SERVER)
```

Comment:
Instead of introducing another proto type, let's just use existing "kind" constants: `types.KindAppServer`, `types.KindDatabaseServer`, etc. https://github.com/gravitational/teleport/blob/v8.0.0/api/types/constants.go#L113